fastlane_core/lib/fastlane_core/build_watcher.rb

```diff
@@ -23,7 +23,7 @@ def wait_for_build_processing_to_be_complete(app_id: nil, platform: nil, poll_in
       private
 
       def watching_build(app_id: nil, platform: nil)
-        processing_builds = Spaceship::TestFlight::Build.all_processing_builds(app_id: app_id, platform: platform)
+        processing_builds = Spaceship::TestFlight::Build.all_processing_builds(app_id: app_id, platform: platform, retry_count: 2)
```

Comment:
Is there a particular reason for having 2 as the default?

Reply:
This is the default value `FastlaneCore::BuildWatcher#matching_build` is [using](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/build_watcher.rb#L33). I kept the same default value but besides of that no other reason.